pymultimatic/model/mode.py

```diff
@@ -66,8 +66,10 @@ class OperatingModes:
     . The newly set temperature will be applied for a certain period of
     time."""
 
+    TIME_CONTROLLED = OperatingMode('TIME_CONTROLLED')
```

Comment:
I found some doc about the time controller: https://www.vaillant.co.uk/downloads/aproducts/controls/sensocomfort/vrc-720-gb-0020287900-00-1714662.pdf (page 3, point 2.7). This mode should be added in `MODES` in `Zone`. This should also be handled in `Function#active_mode` or ine `Zone#active_function`. 

As far as I understand, this is quite similar as `AUTO` mode, the only difference I can see are these "holes" in the time periods